src/sys.d.ts

```diff
@@ -0,0 +1,1352 @@
+  midnight(tz?: TimeZone): DateTime
+  toCode(): string
+}
+export class Enum extends Obj {
+  name$(): string
+  ordinal(): number
+}
+export class Endian extends Enum {
+  static big(): Endian
+  static little(): Endian
+  static vals(): List<Endian>
+  static fromStr(name$: string, checked?: boolean): Endian
+}
+export class Test extends Obj {
+  curTestMethod(): Method
+  setup(): void
+  teardown(): void
+  verify(cond: boolean, msg?: string | null): void
+  verifyTrue(cond: boolean, msg?: string | null): void
+  verifyFalse(cond: boolean, msg?: string | null): void
+  verifyNull(a: JsObj | null, msg?: string | null): void
+  verifyNotNull(a: JsObj | null, msg?: string | null): void
+  verifyEq(a: JsObj | null, b: JsObj | null, msg?: string | null): void
+  verifyNotEq(a: JsObj | null, b: JsObj | null, msg?: string | null): void
+  verifySame(a: JsObj | null, b: JsObj | null, msg?: string | null): void
+  verifyNotSame(a: JsObj | null, b: JsObj | null, msg?: string | null): void
+  verifyType(obj: JsObj, t: Type): void
+  verifyErr(errType: Type | null, c: ((argA: Test) => void)): void
+  verifyErrMsg(errType: Type, errMsg: string, c: ((argA: Test) => void)): void
+  fail(msg?: string | null): void
+  tempDir(): File
+}
+export class TestErr extends Err {
+}
+export class NameErr extends Err {
+}
+export class Duration extends Obj {
+  static defVal(): Duration
+  static minVal(): Duration
+  static maxVal(): Duration
+  static now(): Duration
+  static nowTicks(): number
+  static make(ticks: number): Duration
+  static fromStr(s: string, checked?: boolean): Duration
+  static boot(): Duration
+  static uptime(): Duration
+  ticks(): number
+  negate(): Duration
+  mult(b: number): Duration
+  multFloat(b: number): Duration
+  div(b: number): Duration
+  divFloat(b: number): Duration
+  plus(b: Duration): Duration
+  minus(b: Duration): Duration
+  abs(): Duration
+  min(that: Duration): Duration
+  max(that: Duration): Duration
+  clamp(min: Duration, max: Duration): Duration
+  floor(accuracy: Duration): Duration
+  toMillis(): number
+  toSec(): number
+  toMin(): number
+  toHour(): number
+  toDay(): number
+  toLocale(): string
+  toCode(): string
+  toIso(): string
+  static fromIso(s: string, checked?: boolean): Duration
+}
+export class Process extends Obj {
+  command(): List<string>
+  command(it: List<string>): void
+  dir(): File | null
+  dir(it: File | null): void
+  mergeErr(): boolean
+  mergeErr(it: boolean): void
+  out(): OutStream | null
+  out(it: OutStream | null): void
+  err(): OutStream | null
+  err(it: OutStream | null): void
+  in$(): InStream | null
+  in$(it: InStream | null): void
+  static make(cmd?: List<string>, dir?: File | null): Process
+  env(): Map<string, string>
+  run(): this
+  join(): number
+  kill(): this
+}
+export class Decimal extends Num {
+  static defVal(): number
+  static fromStr(s: string, checked?: boolean): Decimal
+  static negate(self: number): number
```

Comment:
This looks to not have the same methods as the js class, maybe a codegend error on the js side?